web-api/openapi/front_office.yaml

```diff
@@ -172,6 +172,7 @@ components:
             $ref: ./schemas/document.yml#/DocumentPropertyString
           $in:
             $ref: ./schemas/document.yml#/DocumentPropertyArrayString
+            maxItems: 10
```

Comment:
Let's point this out to @acrrd not that the limit is directly conflicting with one of the customers use-cases.

Reply:
i'll bring it up once he is back